.github/workflows/add-release-reviewers.yml

```diff
@@ -8,35 +8,10 @@ on:
 permissions: {}
 
 jobs:
-  add-reviewers:
-    # Tests that label is added on the PR
-    if: ${{ github.event.label.name == 'release' }}
-    runs-on: ubuntu-latest
-    env:
-      ORG: ${{ github.event.organization }}
-      # we can use the team slug when making this a reusable workflow
-      TEAM_SLUG: x
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    permissions:
-      contents: read
-      pull-requests: write
-    steps:
-      - id: get-members
-        run: |
-          DATA=$(gh api \
-            -H "Accept: application/vnd.github+json" \
-            -H "X-GitHub-Api-Version: 2022-11-28" \
-            /orgs/$ORG/teams/$TEAM_SLUG/members?role=maintainer&per_page=100 \
-            | jq 'reduce inputs as $i (.; . += $i)') \
-          echo "data=$DATA" >> $GITHUB_OUTPUT
-
-      # assign reviewers
-      - id: assign-reviewers
-        run: |
-          curl -L \
-            -X POST \
-            -H "Accept: application/vnd.github+json" \
-            -H "Authorization: Bearer $GITHUB_TOKEN" \
-            -H "X-GitHub-Api-Version: 2022-11-28" \
-            https://api.github.com/repos/$ORG/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers \
-            -d '{"reviewers":[${{ join(fromJson(steps.get-members.outputs.data).*.login) }}]}' \
+  add-reviewers-to-release-pr:
+    uses: mui/mui-public/.github/workflows/reusable-add-reviewers-to-pr.yml@2e8907ef6d60b6e60c058c110d61bd07c9ae8c43
+    with:
+      team-slug: x
+      label-name: release
+    secrets:
+      token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You could probably have used `secrets: inherit` and removed the defined input in the workflow, but this is also clearer that the token is required so. 🤷

Reply:
Thats exactly why i added it here. :D

Reply:
technically you could also add another secret here that has a fine grained permission schema instead of the general one